src/backend/TrafficCourts/Messaging/MessageContracts/Class1.cs

```diff
@@ -0,0 +1,110 @@
+﻿using MassTransit;
+using TrafficCourts.Messaging.Models;
+
+namespace TrafficCourts.Messaging.MessageContracts;
+
+public class CheckEmailVerificationTokenRequest
```

Comment:
The filename is Class1.cs.  Was this intentional? Should it be named with this class name instead?

Reply:
True, was overloaded getting the changes out. I will resolve this week after merge.